### ISSUE TITLE
[Dev] V2 Liquidity Sort Algo

### ISSUE THREAD

**Issue by pgebheim** [AugurProject/augur#2764]

**Comment by pacif1c** [comment]
A few thoughts for v2 sort: 

1] currently, I think you can game the sort as follows:

If you’re a market creator, you can add an extra outcome that you know is not going to happen. e.g., Tim Ryan in Democratic Nominee market. Then place a giant ask at just over market creator fees (so it passes spread filter), e.g., at .01 in a market with .95% fees. 

This creates the incentive to 1) add outcomes that nobody wants to trade in 2) add liquidity to those outcomes that nobody wants to trade in rather than to relevant outcomes, in order to prop up a market's rank at no real risk, other than locking up your capital.

2] I think that the sort + spread filter should ignore fees going to *affiliates* up to a certain threshold. So let's say a market has 1% creator fees and half of that is going to affiliates. I think that .5% should *not* be included when calculating the spread. 

In other words, market creators should *not* be penalized for giving a generous allocation to affiliates, if anything they should be rewarded. All else equal, a market with 1% fees with 75% of that going to affiliates should rank higher than a market with 1% fees with 25% of that going to affiliates.  

3] this came up before, but I think the spread filter should require a two-sided order book. This would avoid the issue of dead outcomes propping up a market. e.g., in a "which team will win the Super Bowl" if 6 of 8 listed teams have been eliminated, low asks on those 6 teams with no bids would prop up the market's rank even if it lacks any real/useful liquidity.
